db/migrate/20200707172640_add_engagement_type_to_jobs.rb

```diff
@@ -0,0 +1,5 @@
+class AddEngagementTypeToJobs < ActiveRecord::Migration[5.2]
+  def change
+    add_column :jobs, :engagement_type, :text, null:false, default: ''
```

Comment:
@Robbieakoto let's make the `engagement_type` a hard requirement. for existing posts, we can set all of them to `full_time`. i don't see any problem with that so long as the poster can go back and change the current engagement type. what do you think?